### PR TITLE
#164069176 Install and setup sass

### PR DESCRIPTION
**Pull request by BagzieGracious**
**What does this PR do**
This PR installs and sets up `sass` in the application.

**Description of tasks to be completed**
- Ensure that `sass-loader` and `node-sass` are installed
- Ensure that `sass-loader` and `node-ass` are added in the `package.json` file
- Ensure that sass rules are added in the `webpack.config.js file`

**How can this be tested??**

- Clone the project
   `git clone https://github.com/andela/ah-aquaman-frontend.git`

- Navigate to the project directory
   `cd ah-aquaman-frontend`

-  Install dependencies using `npm install`
- Run Application locally using `npm start`
- Add a sass file and style rules to it

**Any background context you want to provide**
N/A

**What are the relevant pivotal tracker stories**

- [#164069176](https://www.pivotaltracker.com/story/show/164069176)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
 class App extends Component {
-    render() {
-        return (
-          <div>
-              <h1>Authors Haven Application!</h1>
-            </div>
-        );
-    }
+  render() {
```
Comment (CryceTruly) [inline src/App.js]: Thank you for the good work @BagzieGracious , However, this component should be written as a pure function as our eslintrc specifies.You can write it as 
```
const App = () => (
  <div>
    <h1>Authors Haven Application!</h1>

  </div>
);

```

**Review by CryceTruly — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (BagzieGracious) [reply src/App.js]: Thank you @CryceTruly for the review, but I have solved the issue.

**Review by Peace-Apple — APPROVED** [review]
Good work @BagzieGracious

**Review by Peace-Apple — APPROVED** [review]

**Review by CryceTruly — APPROVED** [review]

**Review by wasibani-roy — APPROVED** [review]

File: webpack.config.js [hunk 2]
```diff
@@ -2,29 +2,33 @@ const path = require("path");
...
+      },
+      {
+        test: /\.scss$/,
+        use: ["style-loader", "css-loader", "sass-loader"],
+      },
     ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: "./src/index.html",
```
Comment (AmosWels) [inline webpack.config.js]: Move the index.html file to a public folder outside the src folder and update this path as well from `"./src/index.html"` to `"./public/index.html"`

**Review by AmosWels — CHANGES_REQUESTED** [review]
Good work @BagzieGracious, I've noticed that you've added some more rules in the eslintrc file. Be keen to remove any unnecessary rules that might not be required for this project.

File: webpack.config.js (hunk 2 above)
Comment (BagzieGracious) [reply webpack.config.js]: @AmosWels, I have successfully moved the `index.html` file in the `public` folder.
Thank you

**Review by AmosWels — APPROVED** [review]

**Pull request merged**